db/seeds.rb

```diff
@@ -5,3 +5,26 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+Student.destroy_all
+
+r = Random.new
+
+gender_array = ["male", "female", "transgender", "other"]
+shirt_size = ["xs", "s", "m", "l", "xl"]
+allergy_array = ["peanuts", "eggs", "lactose"]
+
+300.times do
+  Student.create!(
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    grade_level: r.rand(6..8),
+    gender: gender_array[r.rand(gender_array.length)],
+    gpa: r.rand(4.0).round(2),
+    detention_count: r.rand(10),
+    shirt_size: shirt_size[r.rand(shirt_size.length)],
+    food_allergies: allergy_array[r.rand(allergy_array.length)..r.rand(allergy_array.length)],
+    mentor_id: r.rand(10)
```

Comment:
Actually create mentors (teachers) to assign the students to (the models and such exist).